unfoldingWord/translationCore
# Enable or disable developer mode with the konami code

#### This pull request addresses:

This addresses the issue of preventing users from accessing developer mode and messing up their projects.



#### How to test this pull request:

Anywhere in the app (even if you are in a text field!!!) you can input the following command:
Up, Up, Down, Down, Left, Right, Left, Right, B, A
and you should see an alert display to inform you that you have either entered or left developer mode.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/883)
